chat-plugins/othermetas.js

```diff
@@ -310,4 +310,25 @@ exports.commands = {
 		this.sendReply(`|raw|${Chat.getDataPokemonHTML(template)}`);
 	},
 	tiershifthelp: [`/ts OR /tiershift <pokemon> - Shows the base stats that a Pokemon would have in Tier Shift.`],
+
+	'!scalemons': true,
+	scale: 'scalemons',
+	scalemons: function (target, room, user) {
+		if (!this.runBroadcast()) return;
+		if (!toId(target)) return this.parse(`/help scalemons`);
+		let template = Object.assign({}, Dex.getTemplate(target));
+		if (!template.exists) return this.errorReply(`Error: Pokemon ${target} not found.`);
+		let newStats = Object.assign({}, template.baseStats);
+		let stats = ['atk', 'def', 'spa', 'spd', 'spe'];
+		// @ts-ignore
+		let pst = stats.map(stat => template.baseStats[stat]).reduce((x, y) => x + y);
+		let scale = 600 - template.baseStats['hp'];
+		for (const stat of stats) {
+			// @ts-ignore
+			newStats[stat] = Dex.clampIntRange(template.baseStats[stat] * scale / pst, 1, 255);
+		}
+		template.baseStats = Object.assign({}, newStats);
```

Comment:
`newStats` is a copy, so no need to copy it again. Or better still, use the same code as you did in `formats.js`...